src/components/Button/Button.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import Loader from "../Loader";
+import style from "./Button.module.scss";
+import classNames from "classnames";
+
+export type ButtonProps = React.ButtonHTMLAttributes<HTMLButtonElement> & {
+  /** Состояние загрузки */
+  loading?: boolean;
+  /** Текст кнопки */
+  children: React.ReactNode;
+};
```

Comment:
[+] Мы обычно для children используем PropsWithChildren, но это не крит)

```suggestion
export type ButtonProps = React.PropsWithChildren<{
  /** Состояние загрузки */
  loading?: boolean;
}> & React.ButtonHTMLAttributes<HTMLButtonElement>;
```

Reply:
Пон, children во втором случае автоматически добавляется к остальным пропсам и нам не приходится явно его указывать